pkg/storkctl/clusterpair.go

```diff
@@ -33,7 +33,15 @@ const (
 	gcloudBinaryName      = "gcloud"
 )
 
-var clusterPairColumns = []string{"NAME", "STORAGE-STATUS", "SCHEDULER-STATUS", "CREATED"}
+var (
+	clusterPairColumns = []string{"NAME", "STORAGE-STATUS", "SCHEDULER-STATUS", "CREATED"}
+
+	projectMappingHelpString = "Project mappings between source and destination clusters. \n" +
```

Comment:
Do we need to say this is specific to Rancher in the first line itself?

Reply:
Moved (currently supported only for Rancher) to the top